package.json

```diff
@@ -1,45 +1,25 @@
 {
-  "name": "@microsoft.azure/openapi-validator",
-  "version": "0.1.1",
-  "description": "Azure OpenAPI Validator",
-  "main": "src/index.js",
-  "scripts": {
-    "build": "gulp"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Azure/azure-openapi-validator.git"
-  },
-  "keywords": [
-    "azure",
-    "openapi",
-    "swagger",
-    "validation",
-    "validator",
-    "linter"
-  ],
-  "author": "Microsoft Corporation",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/Azure/azure-openapi-validator/issues"
-  },
-  "homepage": "https://github.com/Azure/azure-openapi-validator/blob/master/README.md",
-  "devDependencies": {
-    "@types/js-yaml": "^3.5.30",
-    "@types/jsonpath": "^0.1.29",
-    "@types/node": "^7.0.18",
-    "gulp": "3.9.1",
-    "gulp-run": "1.7.1",
-    "gulp-clean": "0.3.2",
-    "gulp-mocha": "4.3.1",
-    "mocha": "3.2.0",
-    "mocha-typescript": "1.0.22",
-    "typescript": "2.3.3"
-  },
-  "dependencies": {
-    "fs": "^0.0.1-security",
-    "js-yaml": "^3.8.4",
-    "jsonpath": "^0.2.11",
-    "vscode-jsonrpc": "^3.2.0"
-  }
-}
+    "name": "CommonDeps",
```

Comment:
This is simply a package.json for dev dependencies (to install packages like gulp to actually run the build scripts), we will never publish this.